testdata/.tranor/envs.json

```diff
@@ -0,0 +1,14 @@
+[
+	{
+		"name": "dev",
+		"dnsSuffix": "dev.mycompany.com"
```

Comment:
I suggest changing mycompany.com to example.com as it doesn't make it less understandable and it does not use a valid DNS name.